resources/states.raml

```diff
@@ -79,7 +79,58 @@ post:
         oauth_2_0:
           {
             scopes:
-              ['manage_project:{projectKey}', 'manage_orders:{projectKey}'],
+              ['manage_project:{projectKey}', 'manage_states:{projectKey}'],
+          },
+      ]
+    responses:
+      200:
+        body:
+          application/json:
+            example: !include ../examples/state.example.json
+/key={key}:
```

Comment:
Could you please put this block befor the `{ID}:` one?

As the ID block acts like a catch all, tools like vrap would not correctly use this definition